src/App.jsx

```diff
@@ -17,38 +18,6 @@ export const App = () => (
```

Comment:
getUserById function is not being used in the App component. If it's being used elsewhere, move it to a separate utility file.